### PR TITLE
[sc-9072] Bump iOS RadarSDK version to 3.3.1

### PR DESCRIPTION
**Pull request by jrtibbetts**
[[iOS Capacitor] Bump capacitor to iOS 3.3.1](https://app.shortcut.com/radarlabs/story/9072/bump-capacitor-to-ios-3-3-1)

Bumps the RadarSDK dependency to [`3.3.1`](https://github.com/radarlabs/radar-sdk-ios/releases/tag/3.3.1), which adds enhanced logging (in [`3.3.0`](https://github.com/radarlabs/radar-sdk-ios/releases/tag/3.3.0)). There are no other user-facing or breaking changes.

### CODE REVIEW

File: ios/Podfile.lock
Comment (jsani-radar) [inline ios/Podfile.lock:5]: should we be bumping this to 3.3.1 as well?

File: package.json
Comment (jsani-radar) [inline package.json:18]: do we need this dependency? seems like it's causing the massive diff in `package-lock.json`

File: example/capacitor.config.json
Comment (jsani-radar) [inline example/capacitor.config.json:1]: necessary or leftover from your local setup?

**Review by jsani-radar — COMMENTED** [review]
few small comments, besides Brett's changes, it seems like this was as simple as updating the podspec and Podfile?

File: example/capacitor.config.json
Comment (jrtibbetts) [reply example/capacitor.config.json:1]: This was autogenerated, as far as I can tell.

File: ios/Podfile.lock
Comment (jrtibbetts) [reply ios/Podfile.lock:5]: This file was deleted.

**Comment by jrtibbetts** [comment]
Yes--those are the only two manual edits I made. Everything else was changed as part of the build & install process.

File: example/capacitor.config.json
Comment (phil-brown) [reply example/capacitor.config.json:1]: Can you add this to .gitignore instead of committing it?

File: android/src/main/java/io/radar/capacitor/RadarPlugin.java
Comment (phil-brown) [inline android/src/main/java/io/radar/capacitor/RadarPlugin.java:459]: see comments on #26

File: run_example,sh
Comment (phil-brown) [inline run_example,sh:1]: `.sh`?

File: run_example,sh
Comment (phil-brown) [inline run_example,sh:23]: On Android this will run the app on an Android device or emulator, not Android Studio.

**Review by phil-brown — CHANGES_REQUESTED** [review]

File: run_example,sh
Comment (jrtibbetts) [reply run_example,sh:1]: 🤦

File: run_example,sh
Comment (jrtibbetts) [reply run_example,sh:23]: Should I change the documentation, or the command?

File: example/capacitor.config.json
Comment (jrtibbetts) [reply example/capacitor.config.json:1]: Let me delete it and see whether it gets recreated.

File: run_example.sh
Comment (phil-brown) [inline run_example.sh:23]: Can you update comment here? This commands should not open the IDE, it should run the app. Also, the first line I believe is what is needed to ensure the capacitor library stuff is updated on the paths needed for the build. Finally, for Android, I believe this is missing the build step (see circleci config)

Comment (jrtibbetts) [reply run_example.sh]: I just ran it again--it just opens the project Android Studio. For iOS, it opens the project in Xcode.

File: example/capacitor.config.json
Comment (jrtibbetts) [reply example/capacitor.config.json:1]: This file is for configuring the app, and isn't just machine-generated, so it shouldn't be in `gitignore`.

File: run_example,sh
Comment (phil-brown) [reply run_example,sh]: It needs both. I see the documentation change now. But would be good to move this comment down a line as well, since I believe the npx sync is part of the build as well.

Also, can you add a specific input check here, and if it is android, do `cd example ; ./gradlew assemble ; cd ..`?

File: example/capacitor.config.json
Comment (phil-brown) [reply example/capacitor.config.json:1]: Maybe drop the comment? For me, running this command for android allows me to select the device, then launches the app on the emulator or real device. Technically, I have AS running already, so it could just be that it relies on the individual SDKs to perform the actual deploy.

File: run_example.sh
Comment (phil-brown) [reply run_example.sh]: Maybe just drop the comment? For me, it runs on the command line and asks which device to deploy on, then runs the app on that device or emulator. Maybe it is dependent on the IDE? I have mine running basically all the time already.

Comment (jrtibbetts) [reply run_example.sh]: I tried your other suggestion of running `./gradlew assemble` from the `example` directory, but there's no `gradlew` file. At this point, since the script was just meant to be a quick way for me to build and run on iOS, and shouldn't hold up this merge, I'm tempted just to make it iOS-specific.

**Review by tjulien — APPROVED** [review]

File: run_example.sh
Comment (phil-brown) [reply run_example.sh]: @jrtibbetts looks like you removed it again? Yes, if you can, please just make this a `run_ios_example.sh` file instead. I'd prefer to just use Gradle for android anyway.

**Comment by jsani-radar** [comment]
can we merge / rebase this to get it up to date since https://github.com/radarlabs/capacitor-radar/pull/26 was merged?

File: run_example,sh
Comment (jrtibbetts) [reply run_example,sh]: Done. Give it another try.

**Comment by jrtibbetts** [comment]
> can we merge / rebase this to get it up to date since https://github.com/radarlabs/capacitor-radar/pull/26 was merged?

Done.

File: run_example.sh
Comment (jrtibbetts) [reply run_example.sh]: Welp. I fixed it the other day, but had committed it to a different branch. I've now fixed it for both platforms. Even if you still use Gradle directly, I'd like to keep the Android support in the script for when I run and test it.

**Comment by jrtibbetts** [comment]
Awaiting @phil-brown's approval.

File: run_example.sh
Comment (jrtibbetts) [reply run_example.sh]: I removed it from the git repo.

File: .gitignore
Comment (phil-brown) [inline .gitignore:13]: can this be removed now?

**Review by phil-brown — APPROVED** [review]

File: .gitignore
Comment (jrtibbetts) [reply .gitignore:13]: No, because I still use a local copy.

**Pull request merged**
